Classes/PKLog.swift

```diff
@@ -38,5 +38,10 @@ import Log
      PKLog.warning(one, two, three, separator: " - ")
      PKLog.error(error, terminator: "\n")
  */
-public let PKLog = Logger()
+
+public let PKLog: Logger = {
+    let logger = Logger()
+    logger.minLevel = .trace
```

Comment:
@OrenMe this is something that should be done on the app side. we can't control which scheme they are running. they should have configuration that will change logger min level. here this is only as a default settings in case the app doesn't set a log level.